kvbc/include/reconfiguration_add_block_handler.hpp

```diff
@@ -0,0 +1,103 @@
+// Concord
+//
+// Copyright (c) 2018-2021 VMware, Inc. All Rights Reserved.
+//
+// This product is licensed to you under the Apache 2.0 license (the "License").
+// You may not use this product except in compliance with the Apache 2.0 License.
+//
+// This product may include a number of subcomponents with separate copyright
+// notices and license terms. Your use of these subcomponents is subject to the
+// terms and conditions of the subcomponent's license, as noted in the LICENSE
+// file.
+
+#pragma once
+
+#include "reconfiguration/ireconfiguration.hpp"
+#include "db_interfaces.h"
+#include "hex_tools.h"
+#include "block_metadata.hpp"
+
+namespace concord::kvbc::reconfiguration {
+class ReconfigurationHandler : public concord::reconfiguration::IReconfigurationHandler {
+ public:
+  ReconfigurationHandler(kvbc::IBlockAdder& block_adder, kvbc::IReader& ro_storage)
+      : blocks_adder_{block_adder}, block_metadata_{ro_storage} {}
+  bool handle(const concord::messages::WedgeCommand& command, concord::messages::ReconfigurationErrorMsg&) override {
+    std::vector<uint8_t> serialized_command;
+    concord::messages::serialize(serialized_command, command);
+    auto blockId = persistReconfigurationBlock(serialized_command, command.bft_seq_num, reconfiguration_key_prefix + 1);
```

Comment:
I think it will be easier to reason about and also when refactoring/editing if we use constants for the whole key per command.

Reply:
done